api_tests/draft_registrations/views/test_draft_registration_contributor_detail.py

```diff
@@ -0,0 +1,268 @@
+import pytest
+
+from api.base.settings.defaults import API_BASE
+from api_tests.nodes.views.test_node_contributors_detail import (
+    TestContributorDetail,
+    TestNodeContributorOrdering,
+    TestNodeContributorUpdate,
+    TestNodeContributorPartialUpdate,
+    TestNodeContributorDelete
+)
+from osf_tests.factories import (
+    DraftRegistrationFactory,
+    ProjectFactory,
+    AuthUserFactory
+)
+from osf.utils import permissions
+
+
+@pytest.fixture()
+def user():
+    return AuthUserFactory(given_name='Dawn')
+
+
+class TestDraftContributorDetail(TestContributorDetail):
+    @pytest.fixture()
+    def project_public(self, user, title, description, category):
+        # Defining "project public" as a draft reg, overriding TestContributorDetail
```

Comment:
The entanglement of contributors and nodes makes cleaning up the tests to be more reusable a bit more complicated. I agree with you but wanted to bring it up as a byproduct of the node-centric nature of some of our models.

Reply:
makes sense -- maybe we should consider adding a `contributor-basket` to go along with `file-basket` (since both are much more complex than a basic to-many relation)